src/dal/Jalasoft.TeamUp.Resumes.DAL.Interfaces/IRepository.cs

```diff
@@ -14,5 +15,7 @@ public interface IRepository<T>
         public void Update(int id, T updateObject);
 
         public void Delete(int id);
+
+        public IEnumerable<Resume> GetByName(string name);
```

Comment:
the return should be a Generic IEnumerable in this interface due that any repo with any entity can use it

Reply:
Actually this method should be in the ResumesRepository specific domain interface since this is probably not a method any repo would use

Reply:
It's right, done.